test/y2network/routing_test.rb

```diff
@@ -23,8 +23,9 @@
 
 describe Y2Network::Routing do
   subject(:routing) { described_class.new(tables: [table1]) }
-  let(:table1) { Y2Network::RoutingTable.new([route1]) }
-  let(:route1) { double("Y2Network::Route") }
+  let(:table1) { Y2Network::RoutingTable.new(routes) }
+  let(:route1) { double("Y2Network::Route", default?: true) }
```

Comment:
well, to my taste I would expect real Route object here. I think it is better to mock on edges of API or if there is strong reason like hard to use API for testing. But as it is new API, I think it will be better to use API directly with `Y2Network::Route.new`